Sources/Datadog/Core/FeaturesConfiguration.swift

```diff
@@ -181,9 +182,16 @@ extension FeaturesConfiguration {
         )
 
         if configuration.loggingEnabled {
+            let loggingSampler: Sampler?
+            if let loggingSamplingRate = configuration.loggingSamplingRate {
+                loggingSampler = Sampler(samplingRate: loggingSamplingRate)
+            } else {
+                loggingSampler = nil
+            }
```

Comment:
You can do a one-liner here :)
```suggestion
            let loggingSampler = configuration.loggingSamplingRate.map { Sampler(samplingRate: $0) }
```

But it would be even easier if `configuration.loggingSamplingRate` is not optional.